### PR TITLE
thestrad.com

### PR DESCRIPTION
**Pull request by piquark6046**
### To avoid invalid pull requests, please check and confirm following terms

- [X] This is not an ad/bug report;
- [X] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [X] I have performed a self-review of my own changes;
- [X] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [ ] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [x] Social media buttons — share, like, tweet, etc;
- [X] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

### Enter the issue address

 - https://github.com/AdguardTeam/AdguardFilters/issues/140675

### Add your comment and screenshots

```javascript
setInterval(() => {
  Array.from(document.querySelectorAll('#notice .bottom-row button[title="Options"], .footer-policy button[title="Reject All"]')).forEach((e) => {
    try { e.click() } catch {}
  }, 500)
})
```

None of the following rules work well:
```adblock
thestrad.com#%#//scriptlet('trusted-click-element', '#notice .bottom-row button[title="Options"], .footer-policy button[title="Reject All"]', '', '1500')
thestrad.com#%#//scriptlet('trusted-click-element', '#notice .bottom-row button[title="Options"], .footer-policy button[title="Reject All"]', '', '500')
thestrad.com#%#//scriptlet('trusted-click-element', '#notice .bottom-row button[title="Options"], .footer-policy button[title="Reject All"]')
```

### Terms

- [X] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

**Comment by Alex-302** [comment]
Why three rules are required?

**Comment by piquark6046** [comment]
The reason why I attach the three rules is that multiple selector does not work.

**Comment by Alex-302** [comment]
Got it. Maybe a small delay required after each click.

File: AnnoyancesFilter/Cookies/sections/cookies_specific.txt [hunk 1]
```diff
@@ -9765,6 +9765,8 @@ privacy.lecker.de#%#AG_onLoad(function(){if(!document.cookie.includes("euconsent
...
 consent.unilad.com,consent.ladbible.com,consent.gamingbible.co.uk,consent.unilad.co.uk#%#AG_onLoad(function(){if(window.location.href.includes("consentUUID")){var g=new MutationObserver(function(){var b=document.querySelector('button[title*="Accept" i]');b&&(g.disconnect(),b.click())});g.observe(document,{childList:!0,subtree:!0});setTimeout(function(){g.disconnect()},1E4)}});
 !
+thestrad.com#%#//scriptlet('trusted-click-element', '#notice .bottom-row button[title="Options"]')
+thestrad.com#%#//scriptlet('trusted-click-element', '.footer-policy button[title="Reject All"]')
```
Comment (Alex-302) [inline AnnoyancesFilter/Cookies/sections/cookies_specific.txt:9769]: Please add TODO comment with the link to the bug (don't forget to add an empty comment under the rules or move them below to rules with TODO) https://github.com/AdguardTeam/Scriptlets/issues/284

Comment (Alex-302) [reply AnnoyancesFilter/Cookies/sections/cookies_specific.txt:9769]: @piquark6046

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**